webpack/scenes/ContentViews/__tests__/contentViewPage.test.js

```diff
@@ -330,7 +330,7 @@ test('No results message is shown for empty search', async (done) => {
 
   fireEvent.change(getByLabelText(/text input for search/i), { target: { value: query } });
 
-  await patientlyWaitFor(() => expect(getByText(/No matching Content Views found/i)).toBeInTheDocument());
+  await patientlyWaitFor(() => expect(getByText(/No matching content views found/i)).toBeInTheDocument());
```

Comment:
for some reason this test was failing because it was still looking for the capitalized "Content Views"

Reply:
So weird it wasn't failing on master? 😕